apps/demo-fuels/fuels.config.ts

```diff
@@ -0,0 +1,8 @@
+import { createConfig } from 'fuels';
+
+export default createConfig({
+  workspace: './backend',
+  output: './src/backend-api',
+  useBuiltinForc: true,
+  useBuiltinFuelCore: true,
```

Comment:
Great question!

They were optional at first, but I felt the _implicitness_ of it and the criteria for selecting its default value _(`system` vs. `builtin`)_ could get too divisive and interpretative, creating confusion, so I went with _explicitness_ instead.

However, I moved the matter to a more predictable place—_the `fuels init` command_.

https://github.com/FuelLabs/fuels-ts/blob/4bc7dd8936682538c704c1a018f223f37d2e6d8d/packages/fuels/src/cli/commands/init/index.ts#L44-L45

It will do a smart selection of it when generating new configs:

1.  Are system binaries in place?
    1. Use them by default unless overridden by users via:
       - `--use-builtin-forc`
       - `--use-builtin-fuel-core`
1.  System binaries not found?
    1. Instruct users to install The Fuel Toolchain _(as the preferable method)_
    1. While also giving the option to use the `built-in` binaries _(as a fallback option)_
       - _Opt-in via interactive prompts ([here](https://github.com/FuelLabs/fuels-ts/blob/4bc7dd8936682538c704c1a018f223f37d2e6d8d/packages/fuels/src/cli/commands/init/shouldUseBuiltinForc.ts#L19-L23) and [here](https://github.com/FuelLabs/fuels-ts/blob/4bc7dd8936682538c704c1a018f223f37d2e6d8d/packages/fuels/src/cli/commands/init/shouldUseBuiltinFuelCore.ts#L19-L23))_